addons/binding/org.openhab.binding.netatmo/ESH-INF/config/config.xml

```diff
@@ -1,40 +1,36 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <config-description:config-descriptions
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="http://eclipse.org/smarthome/schemas/config-description/v1.0.0"
 	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/config-description/v1.0.0
-        http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
+		http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
 
 	<config-description uri="thing-type:netatmo:bridge">
 		<parameter name="clientId" type="text">
 			<label>Client ID</label>
-			<description>Client ID provided for the application you created on
-				http://dev.netatmo.com/createapp</description>
+			<description>Client ID provided for the application you created on http://dev.netatmo.com/createapp</description>
 			<required>true</required>
-			<advanced>false</advanced>
 		</parameter>
 
 		<parameter name="clientSecret" type="text">
 			<label>Client Secret</label>
 			<description>Client Secret provided for the application you created</description>
 			<required>true</required>
-			<advanced>false</advanced>
 		</parameter>
 
 		<parameter name="username" type="text">
 			<label>Username</label>
 			<description>Your Netatmo API username (email)</description>
 			<required>true</required>
-			<advanced>false</advanced>
 		</parameter>
 
 		<parameter name="password" type="text">
 			<label>Password</label>
 			<description>Your Netatmo API password</description>
 			<required>true</required>
-			<advanced>false</advanced>
 			<context>password</context>
 		</parameter>
-
+					   
```

Comment:
To much whitespaces in several lines?

Reply:
@cweitkamp : I think I spotted the NPE showed upper. Will be corrected in next PR.